debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresOffsetContext.java

```diff
@@ -146,6 +147,21 @@ Long lastCompletelyProcessedLsn() {
         return lastCompletelyProcessedLsn;
     }
 
+    /**
+     * Returns the LSN that the streaming phase should stream events up to or null if
+     * a stopping point is not set. If set during the streaming phase, any event with
+     * an LSN less than the stopping LSN will be processed and once the stopping LSN
+     * is reached, the streaming phase will end. Useful for a pre-snapshot catch up
+     * streaming phase.
+     */
+    Long getStreamingStoppingLsn() {
```

Comment:
@jpechane has concurrent PR where we replace LSN representation from Long to a dedicated `LSN` type. Depending on which PR gets merged first, this one here should change to that new type, too. Just a heads-up, nothing to act on just yet.

Reply:
I saw #1751 was merged and I've added those changes here.

Reply:
Excellent, thanks!